iot-hub/Tutorials/Routing/SimulatedDevice/Program.cs

```diff
@@ -60,23 +56,30 @@ private static void Main(string[] args)
                 //  http://docs.microsoft.com/azure/iot-hub/tutorial-routing
 
                 Console.WriteLine("Routing Tutorial: Simulated device\n");
-                s_deviceClient = DeviceClient.Create(s_iotHubUri, new DeviceAuthenticationWithRegistrySymmetricKey(s_myDeviceId, s_deviceKey), TransportType.Mqtt);
-                SendDeviceToCloudMessagesAsync();
+                s_deviceClient = DeviceClient.Create(s_iotHubUri, 
+                  new DeviceAuthenticationWithRegistrySymmetricKey(s_myDeviceId, s_deviceKey), TransportType.Mqtt);
```

Comment:
you don't want to be checking in hub URIs and device keys.

Try something like [this](https://github.com/Azure-Samples/azure-iot-samples-csharp/blob/master/iot-hub/Samples/device/DeviceReconnectionSample/Program.cs#L24).